Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -2,7 +2,7 @@ Use the Microsoft Teams integration to send messages and notifications to your t
 This integration was integrated and tested with version 1.0 of Microsoft Teams.
 
 **Note:** 
-- Currently, this integration is supported in Cortex XSOAR 8 and up without using an engine.
+- This integration is supported in Cortex XSOAR 8 and up without using an engine.
```

Comment:
```suggestion
- This integration is supported in Cortex XSOAR 8 when not using an engine.
```

Reply:
@ShirleyDenkberg in this note, i'm saying that its supported in xsoar 8 with no need to an engine (but they can use an engine), so i guess "when not" is not the right expression

Reply:
ok